src/app-azure.js

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { SharedMap } from "fluid-framework";
+import { AzureClient, InsecureTokenProvider } from "@fluidframework/azure-client";
+import { vueRenderView as renderDiceRoller } from "./view";
```

Comment:
Curious why Vue is the default?

Reply:
oh, forgot to update the azure one. fixed it in the main app.js